src/main/webapp/assets/modules/salary-report/sr-job-information.js

```diff
@@ -82,6 +82,28 @@ techlooper.directive("srJobInformation", function ($http, validatorService, $tra
         $('.ic-update-info').removeClass('clicked');
       }
 
+      var demandSkillSuggestion = function () {
+        delete scope.state.demandSkills;
+        if (!scope.sr) return;
+
+        var request = {};
+        scope.sr.jobTitle && (request.jobTitle = scope.sr.jobTitle);
+        scope.sr.jobLevelIds && (request.jobLevelIds = vnwConfigService.getJobLevelIds(scope.sr.jobLevelIds));
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.